tests/commands/predict_test.py

```diff
@@ -158,3 +160,60 @@ def predict_json(self, inputs: JsonDict, cuda_device: int = -1) -> JsonDict:
             assert set(result.keys()) == {"span_start_logits", "span_end_logits",
                                           "span_start_probs", "span_end_probs", "best_span",
                                           "best_span_str", "overridden"}
+
+    def test_alternative_file_formats(self):
+        tempdir = tempfile.mkdtemp()
```

Comment:
Does this tempdir ever get cleaned up?

Reply:
looks like none of the tempdirs in this file do, I added cleanups for all of them